### PR TITLE
Update to get proper block versions

### PR DESCRIPTION
**Pull request by alexlyp**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -127,26 +127,22 @@ func updatetemplateInformation(dcrdClient *dcrrpcclient.Client) {
...
-			// Show Red
+			if tipBlockVersionCount >= int(activeNetParams.BlockRejectNumRequired) {
+				templateInformation.BlockVersionSuccess = true
+			}
 			mostPopularBlockVersionCount = tipBlockVersionCount
-			templateInformation.BlockVersionMostPopular = i
-			templateInformation.BlockVersionMostPopularPercentage = toFixed(float64(tipBlockVersionCount)/float64(templateInformation.BlockVersionWindowLength)*100, 2)
 		}
 	}
 	if templateInformation.BlockVersionCurrent == int32(maxVersion) {
```
Comment (chappjc) [inline main.go:148]: still checking `maxVersion` here even though it's set to 3 above?

Comment (chappjc) [reply main.go:148]: sorry to comment if this is a WIP

Comment (alexlyp) [reply main.go:148]: Sort of  WIP, still trying to figure out a clean algo to consistently get the current block version by observing the last 1000 blocks.

Comment (alexlyp) [reply main.go:148]: I'd rather not hard code the current version, but should/could be done programmitcally.

Comment (chappjc) [reply main.go:148]: how about getting the first 4 bytes of `getwork`? should be a uint32 according to Dave w/ current block version

Or do w/ algorithm like you said w/ `getstakeversion [hash] BlockUpgradeNumToCheck` and count total w/ min version

Comment (alexlyp) [reply main.go:148]: Ah definitely both good options.  Going to merge this as to not block karambles work.  But will open an issue to get back to this first thing Monday morning.

Comment (alexlyp) [reply main.go:148]: https://github.com/decred/hardforkdemo/issues/70

**Pull request merged**